src/ray/util/logging.h

```diff
@@ -261,24 +261,20 @@ class RayLog {
   /// This function to judge whether current log is fatal or not.
   bool IsFatal() const;
 
-  /// TODO(hjiang): Having both log directory and filename for `StartRayLog` is confusing,
-  /// at the end of the day should have only filename. Fix all usage in the next PR.
-  ///
+  /// Get filepath to dump log from [log_dir] and [app_name].
+  /// If [log_dir] empty, return empty filepath.
+  static std::string GetLogFilepathFromDirectory(const std::string &log_dir,
+                                                 const std::string &app_name);
+
   /// The init function of ray log for a program which should be called only once.
   ///
   /// \parem appName The app name which starts the log.
   /// \param severity_threshold Logging threshold for the program.
-  /// \param log_dir Logging output directory name.
-  /// \param log_filepath Logging output filepath.
-  /// \param log_rotation_max_size max bytes for of log rotation.
-  /// \param log_rotation_file_num max number of rotating log files.
-  ///
-  /// Both [log_dir] and [log_filepath] are used to determine log output filename; if
-  /// both empty, the log won't output to file, but to stdout. It's illegal to set
-  /// [log_filepath] and [log_dir] at the same time.
-  static void StartRayLog(const std::string &appName,
+  /// \param log_filepath Logging output filepath. If empty, the log won't output to file,
```

Comment:
Q: I think we do log rotations on this log_filepath, so the final filenames are not *exactly* log_filepath but with some changes, right? If so, and since we have custom (non spdlog-default) naming rules, can we document this here and have some examples? e.g. (IIUC)

// Because of log rotations, the logs be saved to log file names with `.<number>` suffixes.
// Example: if log_filepath is /my/path/raylet.out, the output can be  /my/path/raylet.out, /my/path/raylet.out.1 and  /my/path/raylet.out.2

Reply:
Updated.